scripts/index.js

```diff
@@ -0,0 +1,116 @@
+let tempDegree = 70;
+
+const updateSky = () => {
+    const optionSky = document.querySelector('#sky-options').value;
+    const skyContainer = document.querySelector('#sky');
+    let img;
+
+    if (optionSky === '⛅️Cloudy☁️') {
+        img = 'cloudy';
+    } else if (optionSky === '🌈Sunny☀️') {
+        img = 'sunshine';
+    } else if (optionSky === '⛈Rainy☔️') {
+        img = 'raining';
+    } else if (optionSky === '❄️Snowy☃️') {
+        img = 'snowing'
+    }
+
+    skyContainer.textContent = '';
+    const skyContent = document.querySelector('#skyContent');
+    skyContent.classList = `sky-section ${img}`;
+}
+
+
+const updateTempBackground = (currentTemp) => {
+    let color;
+
+    if (currentTemp >= 80) {
+        color = 'summer';
+    } else if (currentTemp >= 70) {
+        color = 'spring';
+    } else if (currentTemp >= 60) {
+        color = 'fall';
+    } else if (currentTemp >= 50) {
+        color = 'winter';
+    }
+
+    const tempContent = document.querySelector('#tempContent');
+    tempContent.classList = `temperature-box ${color}`;
+}
+
+const updateGarden = (currentTemp) => {
+    let img;
+    const gardenContainer = document.querySelector('#ground');
+    let landscape = '';
+    
+    if (currentTemp >= 80) {
+        img = 'desert';
+    } else if (currentTemp >= 70) {
+        img = 'garden-summer';
+    } else if (currentTemp >= 60) {
+        img = 'autumn';
+    } else if (currentTemp >= 50) {
+        img = 'snowfield';
+    }
+
+    gardenContainer.textContent = landscape;
+    const gardenContent = document.querySelector('#gardenContent')
+    gardenContent.classList = `garden-box ${img}`
+}
+
+const updateTemp = (tempDegree) => {
+    const tempDegreeCounter= document.querySelector('#tempDegree');
+    tempDegreeCounter.textContent = tempDegree;
+    updateTempBackground(tempDegree)
+    updateGarden(tempDegree)
+}
+
+const increaseTemp = () => {
+    tempDegree += 1;
+    updateTemp(tempDegree)
+}
+
+const decreaseTemp = () => {
+    tempDegree -= 1;
+    updateTemp(tempDegree)
+}
```

Comment:
Just a note that you can combine these functions with:

```suggestion
const modifyTemp = (increment) => {
    tempDegree += increment;
    updateTemp(tempDegree)
}
```

Then use an anonymous function to call it:

```javascript
increaseTempArrow.addEventListener('click', () => modifyTemp( +1 )  );
```

And something similar to decrease temp.